changelog/undistributed.rst

```diff
@@ -0,0 +1,19 @@
+* Please follow the template we introduced in NOVEMBER.md file.
+* Every parser need to be added under the corresponding feature.
+
+| Module                  | Version       |
+| ------------------------|:-------------:|
+| ``genie.libs.parser``   |               |
+
+--------------------------------------------------------------------------------
```

Comment:
Can you please fix the conflict in the undistributed file?

Reply:
This is the current dev branch code https://github.com/CiscoTestAutomation/genieparser/blob/master/changelog/undistributed.rst

Reply:
@karmoham, conflicts have been resolved.